gui/src/main/java/io/bisq/gui/main/funds/deposit/DepositView.java

```diff
@@ -124,8 +124,10 @@ public void initialize() {
         tableView.setColumnResizePolicy(TableView.CONSTRAINED_RESIZE_POLICY);
         tableView.setPlaceholder(new Label(Res.get("funds.deposit.noAddresses")));
         tableViewSelectionListener = (observableValue, oldValue, newValue) -> {
-            if (newValue != null)
+            if (newValue != null) {
                 fillForm(newValue.getAddressString());
+                UserThread.execute(() -> amountTextField.requestFocus());
```

Comment:
Hm... Maybe better if we rename "execute". But that is used in many places so I would prefer to do that myself to check all and have an atomic commit with only that refactoring.

Reply:
I think it's fine to have an additional method in proper place (GUIUtil) that simply delegates to UserThread if that method's name tells us what's the purpose. Just in case impl changes later.

Reply:
Ok. We should use that only then for the render in next frame use case as that is GUI specific. The UserThread.execute is GUI agnostic and works for headless versions as well.

Reply:
Actually I am not sure if the UserThread.execute is needed at all cases. Can you test the different views if it works without the UserThread.execute as well? I will wait for merging the PR until that is resolved.